superset-frontend/src/explore/exploreUtils.js

```diff
@@ -99,8 +100,8 @@ export function getExploreLongUrl(
     search[key] = extraSearch[key];
   });
   search.form_data = safeStringify(formData);
-  if (endpointType === 'standalone') {
-    search.standalone = 'true';
+  if (endpointType === URL_PARAMS.standalone) {
+    search.standalone = '1';
```

Comment:
Could you use the constant `DashboardStandaloneMode` here as well?

Ditto in the spec files.

Reply:
fixed